lms/templates/base.html.jinja2

```diff
@@ -9,6 +9,19 @@
           href="{{'lms:static/styles/lms.css'|static_path}}">
     <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
           rel="stylesheet">
+
+    <link rel="apple-touch-icon" sizes="152x152"
+          href="{{ 'lms:static/images/apple-touch-icon-152x152.png'|static_path }}">
```

Comment:
I notice this icon is still the old logo. That's true in h as well though 🤷‍♂️

Reply:
I replaced it with one generated from the new logo. I'll deploy it to QA and test how it looks on my phone's home screen